doc/HowTo.md

```diff
@@ -146,27 +146,72 @@ fuse-soc. To use Open Logic through fusesoc, just add open logic as a library:
+version available through _fusesoc-cores_ may be a few releases behind. Add Open Logic directly (see above) if you want
+to be sure to use the latest release.
+
+You should now get one package listed for every area in Open Logic for every release. You can use the areas
+independently (dependencies are modeled in FuseSoC correctly and resolved automatically). You also see the tutorials
+being available and buildable through FuseSoC.
 
 ```shell
 user$ fusesoc core list
 
 Available cores:
 
-Core                                       Cache status  Description
+Core                                             Cache status  Description
 ================================================================================
-open-logic:open-logic:axi:2.0             :      empty : AXI related modules 
-open-logic:open-logic:base:2.0            : downloaded : Basic Circuitry (e.g. FIFOs, CDCs, ...) 
-open-logic:open-logic:intf:2.0            : downloaded : Interfaces (e.g. I2C, synchronizer, SPI, ...) 
-open-logic:tutorials:quartus_tutorial:1.0 :      empty : quartus tutorial for open-logic, targetting DE0-CV board
-open-logic:tutorials:vivado_tutorial:1.0  :      empty : vivado tutorial for open-logic, targetting Zybo Z7-10 board
+open-logic:open-logic-dev:axi:3.0.2             :      local : local files (release plus WIP); AXI related modules ...
+open-logic:open-logic-dev:base:3.0.2            :      local : local files (release plus WIP); Basic Circuitry ...
+open-logic:open-logic-dev:intf:3.0.2            :      local : local files (release plus WIP); Interfaces ...
+open-logic:open-logic:axi:3.0.2                 :      empty : official release (stable); AXI related modules ...
+open-logic:open-logic:base:3.0.2                : downloaded : official release (stable); Basic Circuitry ...
+open-logic:open-logic:intf:3.0.2                : downloaded : official release (stable); Interfaces ...
+open-logic:tutorials-dev:quartus_tutorial:3.0.2 :      local : local files (release plus WIP); quartus tutorial ...
+open-logic:tutorials-dev:vivado_tutorial:3.0.2  :      local : local files (release plus WIP); vivado tutorial ...
+open-logic:tutorials:quartus_tutorial:3.0.2     :      empty : official release (stable); quartus tutorial ...
+open-logic:tutorials:vivado_tutorial:3.0.2      : downloaded : official release (stable); vivado tutorial ...
 
 ```
 
-**Note:** Like for all other tool integrations, Open Logic entities are compiled into the library _olo_ and can be
-instantiated using e.g. `i_fifo : entity olo.olo_base_fifo_sync`.
+Packages are provided in two libraries:
+
+- Use **open-logic:open-logic** for **downloading the stable release from GitHub**
+- Use **open-logic:open-logic-dev** for using the **local files** (the release number mentioned plus all work in
+  progress which is not released)
+  - This option also allows using content of a feature branch that is checked out.
```

Comment:
This is well-written and alleviates my concerns that the user will be confused about how to use Open-Logic with FuseSoC :)